frontend/src/diagram/sprotty/routing/siriusPolylineEdgeRouter.ts

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import {
+  PolylineEdgeRouter,
+  RoutedPoint,
+  SConnectableElement,
+  SParentElement,
+  SRoutableElement,
+  translatePoint,
+} from 'sprotty';
+import { Bounds, Point } from 'sprotty-protocol';
+import { isSiriusRectangleAnchor } from './siriusPolylineAnchor';
+
+export class SiriusPolylineEdgeRouter extends PolylineEdgeRouter {
+  refPosition(bounds: Bounds, ratio: Point): Point {
```

Comment:
Use the type `Ratio` from `Diagram.types.ts` instead of `Point`

Reply:
Done